spec/Body/Body.vspec

```diff
@@ -170,6 +170,26 @@ Lights:
   type: branch
   description: Exterior lights.
 
+Lights.LightSwitch:
+  datatype: string
+  type: actuator
+  allowed: ['OFF', 'POSITION', 'DRL', 'AUTO', 'BEAM']
+  description: Status of the vehicle main light switch.
+  comment: A vehicle typically does not support all alternatives.
+           Which lights that actually are lit may vary according to vehicle configuration and local legislation.
+           OFF is typically indicated by 0.
+           POSITION is typically indicated by ISO 7000 symbol 0456.
+           DRL (if present) indicates Daytime Running Lights.
```

Comment:
is "DRL" a common abbreviation? Otherwise VSS-wise having something more expressive like DAYLIGHT_RUNNING would not hurt us

Reply:
Quite common (see e.g. https://en.wikipedia.org/wiki/Daytime_running_lamp), but for example ISO 7000 use the term "Daytime running lights" so let us use that.